addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -373,6 +373,18 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
         if (!this.$target[0].dataset.model_name) {
             proms.push(this._applyFormModel());
         }
+        this.defaultEmailToValue = "info@yourcompany.example.com";
+        // Get the email_to value from the data-for attribute if it exists. We
+        // use it if there is no value on the email_to input.
+        const formId = this.$target[0].id;
+        const dataForEl = document.querySelector(`[data-for='${formId}']`);
+        if (dataForEl) {
+            this.dataForEmailTo = JSON.parse(dataForEl.dataset.values
+                .replace('False', '""')
+                .replace('None', '""')
+                .replace(/'/g, '"')
```

Comment:
I can move it in a utility function if you prefer. it's the same code in `s_website_form/000.js`

Reply:
Would be ideal if we were to duplicate those lines, as I have a fix coming that I'd like to merge, those lines are terribly wrong, but we never noticed :grimacing: https://github.com/odoo/odoo/pull/124975/files

Reply:
Let's do that then :+1: For the whole thing: retrieving the data-for values etc.